src/main/java/io/github/binaryoverload/JSONConfig.java

```diff
@@ -202,20 +212,30 @@ public JSONConfig(JsonObject object) {
     /**
      * Basic Constructor that sets a JSONObject as well as the path separator
      *
-     * @param object        The object to assign to the config <i>Cannot be null</i>
-     * @param pathSeparator The path separator to be set <i>Cannot be null, empty or any length
-     *                      other than 1</i>
+     * @param object              The object to assign to the config <i>Cannot be null</i>
+     * @param pathSeparator       The path separator to be set <i>Cannot be null, empty or any length
+     *                            other than 1</i>
+     * @param allowedSpecialChars The allowed special characters which can be used on the path. This cannot conflict
+     *                            with the path separator.
      * @throws NullPointerException     if either of the passed arguments are null
      * @throws IllegalArgumentException if the path separator is empty or not a length of 1
      * @since 1.0
      */
-    public JSONConfig(JsonObject object, char pathSeparator) {
+    public JSONConfig(JsonObject object, char pathSeparator, char[] allowedSpecialChars) {
+        this(allowedSpecialChars);
         Objects.requireNonNull(object, "Input is empty!");
         this.object = object;
         setPathSeparator(pathSeparator);
         this.mode = -1;
     }
 
+    private JSONConfig(char[] allowedSpecialCharacters) {
+        this.allowedSpecialCharacters = allowedSpecialCharacters;
+    }
+
+    private JSONConfig() {
```

Comment:
Why...?

Reply:
For the purpose of not needing to always specify special chars. I could just not pass `this()` but it doesn't cost anything extra and still allows a bit more expansion for anything else that needs to be done in all constructors.

Reply:
:+1: